src/luxonis_ml/data/dataset.py

```diff
@@ -167,6 +170,7 @@ def __init__(
         elif self.bucket_storage == BucketStorage.AZURE_BLOB:
             raise NotImplementedError
         else:
+            self.tmp_dir = ".luxonis_tmp"
```

Comment:
Would it make sense making this configurable or allowing it to be project specific? E.g. if I want to store all of the files in some data folder inside my project instead of to tmp_dir?

Reply:
You mean if you want your local storage and remote storage to be always synced (you want to store files in both)?

Reply:
I meant just in general to have local storage at /home/user/Projects/mymodel/data/ for example instead of /home/user/.luxonis_tmp.

Reply:
The temp directory is only meant for remote mode as a place to stage files before syncing to the bucket. In local mode, we have the base_path. This is operating on the assumption that in remote mode you wouldn't _also_ want to store files locally.